### ISSUE TITLE
Analytics module not working in production mode

### ISSUE THREAD

**Issue by xdcha** [nuxt-community/analytics-module#60]
### Version

[2.2.0](https://github.com/analytics-module/releases/tag/2.2.0)

### Reproduction link

[https://codepen.io/d--c/pen/rNNLXRq?editors=0010](https://codepen.io/d--c/pen/rNNLXRq?editors=0010)

### Steps to reproduce

In development mode, with the debug code the script works fine. In production, I do not get any hits. 

```
googleAnalytics: {
   id: "UA-147623000-1"
  }
```

### What is expected ?

Hits in production mode

### What is actually happening?

No hits in production mode. 

### Additional comments?

I did contemplate just leaving it in dev mode `enabled:true` but then the console is riddled with info which I do not wish to be seen by the user. I am using Nuxt v 2.10.1.

Do I need to change the 'Mode' in my Nuxt.config.js? Or shallI just use the Plugin method available on the official docs? https://nuxtjs.org/faq/google-analytics/

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/analytics-module/issues/c47">#c47</a>)</em></sub></div>

**Comment by cksho** [comment]
same here
"@nuxtjs/google-analytics": "^2.2.0", 
"nuxt": "^2.10.0",

**Comment by xdcha** [comment]
Wonder if it is to do with the latest Nuxt?

**Comment by flamner** [comment]
same, 

"nuxt": "^2.9.2",
"@nuxtjs/google-analytics": "^2.2.0",

**Comment by tetreault** [comment]
confirming this isnt working on my end

**Comment by tetreault** [comment]
Really terrible timing

**Comment by maiconpazin** [comment]
Same here.

**Comment by jake-101** [comment]
same here in nuxt 2.10

**Comment by mfrascati** [comment]
Same here. Downgrading nuxt to 2.9.2 works as temporary fix

**Comment by cliffordh** [comment]
BROKEN, even when downgrading to 2.9.2!

**Comment by DispatchCommit** [comment]
Confirmed broken in 2.10, working in 2.9.2

**Comment by manniL** [comment]
Use the "thumbs up" emoji on the actual issue instead of posting "same here", "broken" or similar. Thanks.

**Comment by ryarturogi** [comment]
Not working either nuxt 2.6.3 & nuxtjs/google-analytics 2.2.0 weird because stopped working just 3 days ago.

**Comment by siquick** [comment]
Confirmed same issue here after upgrading to Nuxt 2.10

**Comment by ryarturogi** [comment]
Im using GTM now after i saw this...
[https://www.memory-lovers.blog/entry/2019/06/20/113000](url)

**Comment by sankhagowit** [comment]
In production using 
```js
"nuxt": "^2.10.1",
"@nuxtjs/google-analytics": "^2.2.0" 
```
with the module setting
```js
modules: [
    [
      "@nuxtjs/google-analytics",
      { id: "GA-ID-Here"}
    ]
]
```

I can confirm the google analytics tracking script does not work.

 However, if I change the module setting to
```js
modules: [
    [
      "@nuxtjs/google-analytics",
      { id: "GA-ID-Here"}, debug: { sendHitTask: true }
    ]
]
```
as @manniL showed above the tracking script works again in production.

**Comment by xdcha** [comment]
@manniL @sankhagowit Thanks so much both. I do have it working now. Thanks once more!!! :-)

**Comment by yooneskh** [comment]
adding this resolved it for me too
```
debug: {
  sendHitTask: true
}
```

**Comment by rwwagner90** [comment]
We should not need to manually set `sendHitTask` to true though. It sounds like something in Nuxt 2.10+ changed with regards to determining if we are in dev or prod, so this is no longer sending hits.

**Comment by MatteoGabriele** [comment]
vue-analytics by default sets sendHitTask to true, not sure why in the nuxt module wrapper, this property has been set to undefined. Doesn't make any sense to me

VueAnalytics Nuxt version: https://github.com/nuxt-community/analytics-module/blob/master/lib/module.js#L7
VueAnalytics Original version:  https://github.com/MatteoGabriele/vue-analytics/blob/master/src/config.js#L44

**Comment by yooneskh** [comment]
Yeah i agree

**Comment by ricardogobbosouza** [comment]
In prod mode `sendHitTask` will be set to` true` in next version https://github.com/nuxt-community/analytics-module/commit/9c55a28ca21ab228675c11e8be5c04f343351041

In dev mode `sendHitTask` is set to` false`

> Note: By default, you won't see hits going out in dev mode. For testing analytics in dev mode, you must add the following to nuxt.config.js

https://github.com/nuxt-community/analytics-module#usage

**Comment by isaumya** [comment]
@ricardogobbosouza When will the next version be published? I am using the top-level option so in the meantime should I just do this:

```js
googleAnalytics: {
    id: 'UA-1234567-8',
    debug: {
      sendHitTask: process.env.NODE_ENV !== 'development'
    }
  }
```

**Comment by ricardogobbosouza** [comment]
@manniL can publish a new version?

**Comment by ricardogobbosouza** [comment]
Fixed on v2.2.1

**Comment by manniL** [comment]
(release pending due to lack of publishing rights. should be resolved tomorrow.)

**Comment by ricardogobbosouza** [comment]
I thought I had released

**Comment by pi0** [comment]
Fixed in 2.2.1: https://github.com/nuxt-community/analytics-module/blob/master/CHANGELOG.md#221-2019-10-25

**Comment by msonowal** [comment]
@xdcha 
wo you are using lazysizes, curious to see that, can you show me how are you implementing in the project as I am also trying to use that
